        TestException(String msg, Throwable cause) {
            super(msg, cause);
        }

